src/Misc/layoutbin/en-US/strings.json

```diff
@@ -281,6 +283,7 @@
   "FileDoesNotExist": "File '{0}' does not exist or is not accessible.",
   "FileNotFound": "File not found: '{0}'",
   "FilePathNotFound": "Cannot find the file path for '{0}'.",
+  "FileShareOperatingSystemNotSupported": "File share artifacts are not currently supported on OSX/Linux.",
```

Comment:
>OSX/Linux [](start = 95, length = 9)

[1] non-Windows systems
[2] *NIX/MAC systems

Reply:
Linux means - React OS is allowed :)

---
In reply to: [319298086](https://github.com/microsoft/azure-pipelines-agent/pull/2406#discussion_r319298086) [](ancestors = 319298086)